java-lib/src/main/java/com/wavefront/data/AnnotationUtils.java

```diff
@@ -0,0 +1,53 @@
+package com.wavefront.data;
+
+import javax.annotation.Nullable;
+import java.util.Collection;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import com.google.common.collect.ImmutableSetMultimap;
+import com.google.common.collect.Multimap;
+import com.google.common.collect.SetMultimap;
+
+import wavefront.report.Annotation;
+
+/**
+ * Utility methods for working with {@code List&lt;Annotation&gt;}.
+ *
+ * @author vasily@wavefront.com
+ */
+public abstract class AnnotationUtils {
+  private AnnotationUtils() {
+  }
+
+  @Nullable
+  public static SetMultimap<String, String> toMultimap(@Nullable Collection<Annotation> annotations) {
+    if (annotations == null) return null;
```

Comment:
sometimes I find that returning empty multimaps to be easier to reason (for callee)